sdk/core/azure-core/CMakeLists.txt

```diff
@@ -78,10 +78,10 @@ set(
     inc/azure/core/operation.hpp
     inc/azure/core/operation_status.hpp
     inc/azure/core/platform.hpp
-    inc/azure/core/response.hpp
     inc/azure/core/uuid.hpp
     inc/azure/core/version.hpp
     inc/azure/core.hpp
+    inc/azure/response.hpp
```

Comment:
Unless I missed some new discussion/conclusion, I had discussed this with @RickWinter previously, and we landed on the conclusion that the header paths should continue to be under `azure/core` to be consistent with package name.

So, even though we are updating namespaces, keep the header files on disk where they already are.

We want the top level headers to not be noisy in intellisense, and the customer should just include `azure/core.hpp`